packages/storybook/stories/va-language-toggle.stories.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment } from 'react';
+import { getWebComponentDocs, StoryDocs, propStructure } from './wc-helpers';
+import { VaLanguageToggle } from '@department-of-veterans-affairs/web-components/react-bindings';
+
+const languageToggleDocs = getWebComponentDocs('va-language-toggle');
+
+export default {
+  title: 'Components/Language Toggle',
+  id: 'components/va-language-toggle',
+  parameters: {
+    componentSubtitle: 'va-language-toggle web component',
+    docs: {
+      page: () => <StoryDocs storyDefault={Default} data={languageToggleDocs} />,
+    },
+  },
+};
+
+const url = new URL(window.parent.location.href);
+url.searchParams.set('path', '/docs/components-va-language-toggle--docs');
+
+const defaultArgs = {
+  enHref: url.href,
+  esHref: url.href,
+  tlHref: url.href,
```

Comment:
@it-harrison Awesome! Thanks for figuring that out! I think the only other thing that needs to be done is passing in an empty arguments object to the `Template` function so that the code examples populate:

![Screenshot 2024-11-07 at 8 42 32 AM](https://github.com/user-attachments/assets/dd7d2337-207d-4339-8e8f-320bcd58b54b)

Maybe something like this [here on this line](https://github.com/department-of-veterans-affairs/component-library/blob/c24f1cf704d0e9d68bb75204c56b9856f07c4fa9/packages/storybook/stories/va-language-toggle.stories.jsx#L21)? I can't exactly remember what has fixed this in the past.

```js
const Template = ({}) => {
 ...
}
```

Reply:
@jamigibbs - good catch, fixed!